src/callkit.cs

```diff
@@ -47,7 +47,9 @@ public enum CXErrorCodeIncomingCallError : long {
 		Unentitled = 1,
 		CallUuidAlreadyExists = 2,
 		FilteredByDoNotDisturb = 3,
-		FilteredByBlockList = 4
+		FilteredByBlockList = 4,
+		[iOS (17, 0), Watch (10, 0), MacCatalyst (17, 0)]
```

Comment:
IIRC error enums don't take availability attributes on individual enum values.

Reply:
That is so very true. Mea culpa specially when I mentioned in a previous pr.